flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyTestUtil.java

```diff
@@ -162,6 +168,37 @@ static NettyConfig createConfig(int segmentSize, Configuration config) throws Ex
 				config);
 	}
 
+	// ---------------------------------------------------------------------------------------------
+	// Encoding & Decoding
+	// ---------------------------------------------------------------------------------------------
+
+	@SuppressWarnings("unchecked")
```

Comment:
this can also be removed together with https://github.com/apache/flink/pull/7368/files#r389460526

Reply:
Removed the warning